SwrveSDKCommon/Common/SwrveCommon.h

```diff
@@ -0,0 +1,81 @@
+#import <UIKit/UIKit.h>
+
+/*! Swrve SDK shared protocol (interface) definition */
+@protocol ISwrveCommon <NSObject>
+
+-(NSData*) getCampaignData:(int)category;
+-(void) sendQueuedEvents;
+-(int) eventInternal:(NSString*)eventName payload:(NSDictionary*)eventPayload triggerCallback:(bool)triggerCallback;
+-(int) userUpdate:(NSDictionary*)attributes;
+-(void) setLocationVersion:(NSString*)locationVersion;
+-(BOOL) processPermissionRequest:(NSString*)action;
+
+@end
+
+@interface SwrveCommon : NSObject
+
++(id<ISwrveCommon>) getSwrveCommon;
```

Comment:
In the regular sdk we use this syntax:
Swrve *swrve = [Swrve sharedInstance];

Should we be doing similar syntax here also?
[SwrveCommon sharedInstance]